src/Microsoft.AspNetCore.Mvc.Razor/Internal/CompositeTagHelperDescriptorResolver.cs

```diff
@@ -10,22 +10,18 @@ namespace Microsoft.AspNetCore.Mvc.Razor.Internal
     public class CompositeTagHelperDescriptorResolver : ITagHelperDescriptorResolver
```

Comment:
We will need to remove this because `ITagHelperDescriptorResolver` is being removed